web-analogs.md

```diff
@@ -162,8 +162,11 @@ var container = new Container( // grey box
 ### Setting Container Width
 
 To specify the width of a [Container](https://docs.flutter.io/flutter/widgets/Container-class.html)
-widget you set its ```width``` property. This is equivalent to specifying the CSS 
-max-width property.
+widget you set its ```width``` property. This is a static width, unlike the 
```

Comment:
Is there another word, other than "static" ? That could be confused with the language's semantics for "static".